src/utils/misc.ts

```diff
@@ -0,0 +1,10 @@
+/** Truncates a string if it exceeds `length` and inserts `endStr` at the end (empty string to disable) */
+export function truncStr(str: string, length: number, endStr = "...")
+{
+    return str.length > length ? str.substring(0, length) + endStr : str;
+}
+
+export function truncField(content: string, endStr = "...")
+{
+    return truncStr(content, 1000, endStr);
```

Comment:
Wouldn't this need to be `1000 - endStr.length`?

Reply:
max is 1024 chars so no

Reply:
what if i want the endStr to be `123456798123456798123456798123456798123456798123456798`